packages/core/src/axis-chart.ts

```diff
@@ -96,9 +96,15 @@ export class AxisChart extends Chart {
 			"data",
 			"loading"
 		);
-		const isLegendEnabled =
+		let isLegendEnabled =
 			this.model.getOptions().legend.enabled !== false && !isDataLoading;
 
+		const hasExternalLegend = this.model.getOptions().legend?.external
+			?.reference;
+
+		// disable internal legend if external legend is provided
+		isLegendEnabled = !hasExternalLegend;
+
```

Comment:
```suggestion
		const options = this.model.getOptions();
		const hasExternalLegend = Tools.getProperty(options, "legend", "external", "reference");
		// enable the internal legend if user did not provide external legend ref and data is not loading
		const isLegendEnabled = !hasExternalLegend && Tools.getProperty(options, "legend", "enabled") !== false && !isDataLoading;

```


makes me wonder, why not allow the user to have an **external legend and one internal legend** within the chart container? since we have a configuration for legend.enabled, why not allow the user to have an external chart ref, but also put the responsibility on them to **disable the internal legend or keep it**? Personally id like to have options for both an internal and external legend, as well as having one or the other